src/components/connect/NetworkSelect.jsx

```diff
@@ -67,7 +70,7 @@ export default function NetworkSelect() {
           <Box fluid align="center" direction="vertical" mb="6px">
             {networks.map((props) => {
               const { label, value, avatar, description, chainId } = props;
-
+              const { data } = useMemberSummary(address, chainId);
```

Comment:
Calling hooks inside loops is a bit of an anti pattern, if the length of `networks` changes over re-renders then the app will break. Obvs networks is constant here but better to not use hooks like this. Instead create a new component `NetworkSelectOption` and encapsulate the logic there. This should be a new file in `components` and any CSS that is in `NetworkSelect.scss` that is relevant to this component to `NetworkSelectOption.scss` and rename the classes so they follow the convention.